muxy_toast_demo/src/libs/muxy-plugin.js

```diff
@@ -0,0 +1,22 @@
+/*
+  This vuejs plugin loads up the Muxy SDK and automatically binds the Muxy
+  object and a new SDK to the vue global scope
+*/
+import Muxy from '@muxy/extensions-js';
+
+export default {
+  install(Vue, options = {}) {
+    if (!options.extension_id) {
```

Comment:
To avoid confusion, can you change this to `client_id` to match its usage elsewhere?